File: LAB 104/212_contest/H.py
t = int(input())

for i in range(0, t):
    s = input()
    a = b = c = 0
    for m in range(0, len(s)):
        if (s[m] == "A"):
            a = a+1
        elif (s[m] == "B"):
            b = b+1
        else:
            c = c+1


    if (int(a+c) == int(b)):
        print("YES")
    else:
        print("NO")

File: LAB 104/212_contest/H.c
#include <stdio.h>
#include <string.h>

int main() {
    int N, l, a=0, b=0, c=0;
    char *inputStr;
    scanf("%d", &N);

    while (N--) {
        scanf("%s", &inputStr);
        l = strlen(inputStr);
        for (int i=0; i<l; i++) {
            if (inputStr[i] == 'A') {
                a++;
            }
            else if (inputStr[i] == 'B') {
                b++;
            }
            else {
                c++;
            }
        }
        if ((a+c) == b) {
            printf("YES\n");
        }
        else {
            printf("No\n");
        }
        // strncpy(inputStr, "");
    }

    return 0;
}